src/List.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {
+  render,
+} from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  const setup = ({ tasks }) => {
+    const utils = render(<List tasks={tasks} />);
+    return { ...utils };
+  };
```

Comment:
이 부분도 조금 더 간단하게 표현가능 할 것 같아요~
```suggestion
  const handleClickDelete = jest.fn();

  const renderList = (tasks) => render((
    <List
      tasks={tasks}
      onClickDelete={handleClickDelete}
    />
  ));
```